packages/colors/src/colors.js

```diff
@@ -247,6 +247,9 @@ export const warmGray = {
   100: warmGray100,
 };
 
+// Deprecated ☠️
+export const yellow20 = '#fdd13a';
```

Comment:
Do you want to defer this removal until next major?

Reply:
Should we remove the deprecation notice ? I'm unsure otherwise how to add both colors since yellow 30 is part of the updated palette. unless we want to leave the name as yellow20 with the new hex @shixiedesign ?

Reply:
This code should be safe, since there is only one "yellow" we should expose `yellow30` as yellow. `yellow20` is for backwards compat 👍

Reply:
I thought the lines were removed instead of added. Sorry for the confusion.

Reply:
Yeah the new yellow is `yellow30`. Sounds like you guys worked out how to handle this part.